src/ocamlformat.ml

```diff
@@ -15,7 +15,7 @@
 let impl : _ Translation_unit.t =
   let parse = Translation_unit.parse Migrate_ast.Parse.implementation in
   { parse
-  ; input= parse Location.none.loc_start.pos_fname
+  ; input= (fun ~input_name ic -> parse ~input_name ic)
```

Comment:
Would it be clearer to use `parse ?warn:None` here instead? Or maybe there is some other reason to prefer the eta-expansion?

Reply:
Agreed, I'd prefer ``parse ?warn:None``.  Thank you.  I'd be grateful if you want to polish the PR further, otherwise I'd push an update an Monday.  Thanks!